build/test.js

```diff
@@ -1,7 +1,9 @@
+'use strict';
+
 const Builder = require('./Builder.js');
 
-let build = new Builder();
-console.log("Created builder :)");
+const build = new Builder();
```

Comment:
Guess this depends on your definition of a constant ;)

Reply:
Haha ;)